lib/Resources/config/query_types.yml

```diff
@@ -0,0 +1,24 @@
+parameters:
+
+services:
```

Comment:
`public: false` on all services :)

Reply:
Added in https://github.com/netgen/ezplatform-site-api/pull/70/commits/96eea81268c31c3a91bcc77dd39bc9c6a712cbeb